awsutil/awsutil.go

```diff
@@ -56,6 +56,6 @@ func TimeDifference(server string) (time.Duration, error) {
 		return time.Duration(0), errors.New("invalid ntpdate output")
 	}
 
-	f, err := strconv.ParseFloat(string(submatched[1]), 64)
+	f, _ := strconv.ParseFloat(string(submatched[1]), 64)
```

Comment:
I forgot to check the error. I think it will never occur unless ntpdate changes the output. But it should be checked just in case.